### PR TITLE
Reorg accounts page and migration to Advanced

### LINKED ISSUE #3804: Reorg Accounts Page
**Problem**
- The page contains advanced information (such as hashing, and account derivation) that increases its size and could scare the average reader.
- Some chain data are not fetched

**Potential solution**
I propose to migrate the advanced information to an advanced page, as we did for the staking page. I also propose to reorganise the content a bit.

### PR DESCRIPTION
**Pull request by filippoweb3**
PR focused on reorganising the Accounts page, and moving advanced sections on the separate page `learn-account-advanced`

### CODE REVIEW

**Comment by filippoweb3** [comment]
@DrW3RK in this PR, I tried to keep the Basic Account pages as straightforward as possible and moved advanced info to the Advanced menu. I also created a multisig dedicated page as I think it makes the Account page less heavy. Let me know what you think. I followed a similar procedure as for the Staking page.

File: docs/learn/learn-account-advanced.md [hunk 1]
```diff
@@ -0,0 +1,340 @@
+---
+id: learn-account-advanced
+title: Polkadot Accounts Advanced
```
Comment (profradha) [inline docs/learn/learn-account-advanced.md]: Just thinking of a better title than Polkadot Accounts Advanced

```suggestion
title: Polkadot Accounts In-Depth
```

File: docs/learn/learn-account-advanced.md [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+sidebar_label: Accounts
```
Comment (profradha) [inline docs/learn/learn-account-advanced.md:4]:
```suggestion
sidebar_label: Accounts In-Depth
```

File: docs/learn/learn-account-multisig.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
+---
+id: learn-account-multisig
+title: Multi-signature accounts
```
Comment (profradha) [inline docs/learn/learn-account-multisig.md]:
```suggestion
title: Multi-Signature Accounts
```

File: docs/learn/learn-account-multisig.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+sidebar_label: Multi-signature accounts
```
Comment (profradha) [inline docs/learn/learn-account-multisig.md]:
```suggestion
sidebar_label: Multi-Signature Accounts
```

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**